pkg/chunk/table_manager.go

```diff
@@ -47,8 +47,8 @@ func newTableManagerMetrics(r prometheus.Registerer) *tableManagerMetrics {
 
 	m.tableCapacity = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: "cortex",
-		Name:      "dynamo_table_capacity_units",
-		Help:      "Per-table DynamoDB capacity, measured in DynamoDB capacity units.",
+		Name:      "table_capacity_units",
```

Comment:
We discussed this in #2307, it is actually dynamodb specific.

Reply:
https://cloud-native.slack.com/archives/CCYDASBLP/p1585577302112200

> @bboreham: Today it is only on DynamoDB, but it could be implemented for another DB, e.g. CosmosDB at Azure.

Reply:
Roger.  I don't mind either way.

Reply:
If we keep this change, it should be mentioned in the changelog, otherwise will be difficult to notice for the final user.